lib/ohai/plugins/init_package.rb

```diff
@@ -20,12 +20,6 @@
   provides "init_package"
 
   collect_data(:linux) do
-    package_name = nil
-
-    if File.exists?("/proc/1/comm")
-      package_name = File.open("/proc/1/comm").gets.chomp
-    end
-
-    init_package package_name
+    init_package File.exists?("/proc/1/comm") ? File.open("/proc/1/comm").gets.chomp : 'init'
```

Comment:
any reason you can't/shouldn't just set `package_name` to `init` up top and leave the rest of the code untouched? I think it's probably a fair chunk more readable.